agent/agent_test.go

```diff
@@ -14,11 +14,15 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+<<<<<<< HEAD
+	"io"
+=======
 	"github.com/hashicorp/consul/agent/grpc-external/limiter"
 	"github.com/hashicorp/consul/agent/proxycfg"
 	"github.com/hashicorp/consul/agent/proxycfg-sources/local"
 	"github.com/hashicorp/consul/agent/xds"
 	proxytracker "github.com/hashicorp/consul/internal/mesh/proxy-tracker"
+>>>>>>> main
```

Comment:
Looks like a merge conflict was missed here?

Reply:
Fixed, sorry about that.